OSOE-430: Support a single NE instance per solution on Linux, too, in Lombiq.NodeJs.Extensions

sarahelsaig opened the pull request:
[OSOE-430](https://lombiq.atlassian.net/browse/OSOE-430)

[OSOE-430]: https://lombiq.atlassian.net/browse/OSOE-430?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

sarahelsaig committed:
Remove Linux excluding condition.

sarahelsaig committed:
Properly notify if get-path fails.

sarahelsaig committed:
Use environment variable instead of process call.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

sarahelsaig committed:
Use the getCwd() method and ensure the symlinks are not resolved.

sarahelsaig committed:
use getCwd() instead of process.cwd().

sarahelsaig committed:
submodule

sarahelsaig committed:
Use path.join where applicable.

sarahelsaig committed:
Fix require path.

sarahelsaig committed:
typo

sarahelsaig committed:
Improve stability directly after a page load.

sarahelsaig committed:
Fix typo.

sarahelsaig committed:
Fix doc.

sarahelsaig committed:
Improve error reporting.

sarahelsaig committed:
Fix another edge case.

sarahelsaig committed:
Handle SRC=! in the package.json instead.

sarahelsaig committed:
Typo.

sarahelsaig committed:
I temporarily added 10x to every "timeout-minutes", this should be reverted once the problem is diagnosed.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

sarahelsaig committed:
Revert " I temporarily added 10x to every "timeout-minutes", this should be reverted once the problem is diagnosed."

This reverts commit c36d4af022272ac0029cf03ba6ff89b408188717.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

sarahelsaig committed:
More diagnostics.

sarahelsaig committed:
Typo

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

# Conflicts:
#	test/Lombiq.UITestingToolbox

sarahelsaig committed:
Look for the project root instead of the current directory.

sarahelsaig committed:
Add and use get-project-directory.

sarahelsaig committed:
Export project path as environment variable.

sarahelsaig committed:
Relative path only on Windows.

sarahelsaig committed:
Print more information if copy fails.

sarahelsaig committed:
Make directoryToCopy an absolute path from the project directory.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

sarahelsaig committed:
Fix copy-assets.

sarahelsaig committed:
Fix Lombiq.VueJs process-helpers.js

sarahelsaig committed:
Resolve ESlint warnings in Lombiq.NodeJs.Extensions/scripts/lint-styles.js

sarahelsaig committed:
submodule

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

# Conflicts:
#	test/Lombiq.UITestingToolbox

sarahelsaig commented:
Disregard the failed test, only the "Automatically Done Jira Issue" step failed. (known issue https://github.com/Lombiq/GitHub-Actions/issues/130) Looks like I accidentally added the `done-jira-issue-if-checks-succeed` tag a second time when the Jira was already on Done or something like that.

dministro reviewed (commented):
Merge from dev please.

sarahelsaig committed:
Use resolve here instead.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

# Conflicts:
#	Directory.Build.props

sarahelsaig committed:
possibly breaking change

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-430-new

# Conflicts:
#	src/Modules/Lombiq.VueJs
#	test/Lombiq.UITestingToolbox

sarahelsaig committed:
dummy commit to see if the "terser" thing is recurring

dministro committed:
Updating submodules:
 - Lombiq.VueJs: fast-forwarding
 - Lombiq.NodeJs.Extensions: fast-forwarding
 - Lombiq.UITestingToolbox: fast-forwarding
